core-common/src/main/java/org/zalando/nakadi/repository/zookeeper/ZooKeeperHolder.java

```diff
@@ -85,13 +85,15 @@ public CuratorFramework getCuratorFramework() {
 
     public static class StaticCuratorFramework extends CloseableCuratorFramework {
```

Comment:
Please, change name from `StaticCuratorFramework` to `DeafultSubscriptionCurator` or something else more appropriate.